polytope_server/common/queue/sqs_queue.py

```diff
@@ -0,0 +1,84 @@
+import json
+import logging
+from . import queue
+import boto3
+import os
+
+
+class SQSQueue(queue.Queue):
+    def __init__(self, config):
+        host = config.get("host", "localhost")
+        queue_name = config.get("name", "default")
+        region = config.get("region", "eu-central-2")
+        self.keep_alive_interval = config.get("keep_alive_interval", 60)
+        self.visibility_timeout = config.get("visibility_timeout", 120)
+        self.queue_url = "{}/{}".format(host, queue_name)
+
+        logging.getLogger("sqs").setLevel("WARNING")
+        session = boto3.Session(
+            aws_access_key_id=os.getenv("POLYTOPE_S3_ACCESS_KEY"),
```

Comment:
Do we want to reuse the well known names of the boto3 client? Then we don't need to setup the session manually. 
- `AWS_ACCESS_KEY_ID`
- `AWS_SECRET_ACCESS_KEY`

https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html#configuring-credentials